code/modules/mob/living/silicon/ai/freelook/cameranet.dm

```diff
@@ -76,12 +77,16 @@ var/datum/cameranet/cameranet = new()
 // Removes a camera from a chunk.
 
 /datum/cameranet/proc/removeCamera(obj/machinery/camera/c)
+	cameras -= c
```

Comment:
Now that I think of it, maybe it's not needed to concretely remove it from the list, and instead just update the chunk
I think I was very confused tonight and didn't fully understood what I was doing so here's a comment to remember